xcp-ng.repo

```diff
@@ -14,6 +16,20 @@ gpgcheck=1
 repo_gpgcheck=1
 gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-xcpng
 
+
+# The four repositories below this line are involved in the update build and testing process.
+# The typical flow is: incoming -> ci -> testing -> candidates -> updates
+
+# Updates set for imminent xcp-ng-updates publication upon completion of final testing.
+[xcp-ng-candidates]
```

Comment:
"candidates" finally reads quite long, maybe "rc" would be clear enough?

Reply:
Too late :D